server/routes/request.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  loadRequests,
+  createRequest,
+  approveRequest,
+} = require("../controllers/request");
+
+router.route("/").get(protect, loadRequests);
+
+router.route("/create").post(protect, createRequest);
```

Comment:
```suggestion
router.route("/").post(protect, createRequest);
```
We can simplify these routes as the HTTP verbs indicate the purpose